test/e2e/network_test.go

```diff
@@ -166,7 +190,8 @@ func testNorthSouthNetworking(t *testing.T) {
 		}
 	}
 	require.NoError(t, err, "could not create Windows Server deployment")
-	defer testCtx.deleteDeployment(winServerDeployment.GetName())
+	defer testCtx.getDeploymentLogs(winServerDeployment)
```

Comment:
if you think deployments are getting deleted before they are collected, you can defer only deleting of deployment, and do the getDeployment sequentially at the end of the function.

Reply:
Also keep in mind that the webserver deployments don't generate much output and is typically a one liner.